tutorials/dataframe/df001_introduction.C

```diff
@@ -138,7 +138,7 @@ int df001_introduction()
    // Often, operations need to be carried out on quantities calculated starting
    // from the ones present in the columns. We'll create in this example a third
    // column the values of which are the sum of the *b1* and *b2* ones, entry by
-   // entry. The way in which the new quantity is defined is via a runable.
+   // entry. The way in which the new quantity is defined is via a run-able.
```

Comment:
here and below, "runable" could/should be changed to "callable", but not "run-able"

Reply:
done